src/styles/pages/_lte.scss

```diff
@@ -9,15 +11,11 @@ body.lte {
 
   .lte-title {
     display: flex;
-    position: relative;
-    left: 50%;
-    right: 50%;
-    margin-left: -50vw;
-    margin-right: -50vw;
-    width: 100vw;
-    height: 130px;
     justify-content: center;
     align-items: center;
+    width: 200%;
+    margin: 0 -25%;
```

Comment:
It looks like `margin: 0 -25%` may be causing problems when the browser window is resized, because the width of the containing `<div class="row">` is computed differently based on whether the page loads at a certain width versus when the page is resized to that width.

I'm not sure if the fix is to change how the `<div class="row">` width is computed, or if it's to go with a different approach to set the title margins here.

Here are some steps to illustrate the problems I'm seeing:

1. Load page in a desktop-width browser window (~1600px)
2. Reduce browser window to mobile width (~500px). Result: (Possible bug) The blue block behind the title doesn't fill the full width of the page, and the content below the title is very narrow, with wide margins on either side. 
3. Refresh the page. Result: The title looks good. It is scaled well, and the blue block fills the width of the page. The content also fills the full screen width.
4. Expand page to desktop-width. Result: (Possible bug) The text in the title is wider than it was when I loaded the page in step one. (It is on one line, not two.)
5. Refresh the page: Result: The title wraps correctly to two lines. (Possible bug) On Chrome (but not Firefox), I also see a white gap appear between the right edge of the footer and the window.

Reply:
Thanks for the detailed comments and QA @technicalex. I believe all these issues (including needing `overflow-x: hidden;`) was due to the placement of this title within the main container, rather than the body, which required this hacky styling to expand it beyond its parent's width. 

This is all solved by simply moving the title into `wrapper` like so:
https://github.com/MoveOnOrg/ak-template-set/pull/1043/commits/56bd79dbcb8efff9b30dc45101dd86a5a80f1098

I believe this is the type of hero title that is intended to be used on most other pages types as well, so putting it here will make it easy to cut over other page types when they are ready to implement the redesign.